### PR TITLE
Show header in terms of service so user can change language

### PR DESCRIPTION
**Pull request by ngocdh236**

### CODE REVIEW

File: frontend/benefit/applicant/src/components/termsOfService/TermsOfService.tsx [hunk 1]
```diff
@@ -30,10 +30,11 @@ const TermsOfService: React.FC<TermsOfServiceProps> = ({
...
     <Container>
       <FormSection withoutDivider>
-        <$GridCell $colSpan={3} style={{ marginTop: theme.spacingLayout.l }}>
-          <Logo language={locale as LogoLanguage} size="large" />
-        </$GridCell>
-        <$GridCell $colStart={1} $colSpan={12}>
+        <$GridCell
+          $colStart={1}
+          $colSpan={12}
+          style={{ marginTop: theme.spacingLayout.l }}
```
Comment (nenonja) [inline frontend/benefit/applicant/src/components/termsOfService/TermsOfService.tsx:36]: I would prefer css prop: https://styled-components.com/docs/api#css-prop

**Review by melniiv — APPROVED** [review]

File: frontend/benefit/applicant/src/components/termsOfService/TermsOfService.tsx (hunk 1 above)
Comment (ngocdh236) [reply frontend/benefit/applicant/src/components/termsOfService/TermsOfService.tsx]: I have resolved

File: frontend/benefit/applicant/src/components/termsOfService/TermsOfService.tsx [hunk 2]
```diff
@@ -30,14 +30,21 @@ const TermsOfService: React.FC<TermsOfServiceProps> = ({
...
       <FormSection withoutDivider>
-        <$GridCell $colSpan={3} style={{ marginTop: theme.spacingLayout.l }}>
-          <Logo language={locale as LogoLanguage} size="large" />
-        </$GridCell>
-        <$GridCell $colStart={1} $colSpan={12}>
+        <$GridCell
+          $colStart={1}
+          $colSpan={12}
+          css={`
+            margin-top: var(--spacing-layout-l);
```
Comment (nenonja) [inline frontend/benefit/applicant/src/components/termsOfService/TermsOfService.tsx:37]: you can still use theme: `margin-top: ${theme.spacingLayout.l};`

Comment (nenonja) [reply frontend/benefit/applicant/src/components/termsOfService/TermsOfService.tsx:37]: or even: `css={{ marginTop: theme.spacingLayout.l }}` might work

**Review by nenonja — APPROVED** [review]

**Pull request merged**